packages/dapp-svelte-wallet/api/src/lib-wallet.js

```diff
@@ -72,7 +73,44 @@ export function makeWallet({
   myAddressNameAdmin,
   pursesStateChangeHandler = noActionStateChangeHandler,
   inboxStateChangeHandler = noActionStateChangeHandler,
+  localTimerService,
 }) {
+  /**
+   * The current timestamp, in milliseconds (if it is known).
+   *
+   * @type {number | undefined}
+   */
+  let nowMs;
+
+  // Subscribe to the timer service to update our stamp.
+  if (localTimerService) {
+    observeNotifier(E(localTimerService).makeNotifier(0n, 1000n), {
```

Comment:
Are these arguments in nanoseconds? So it updates every 1 millisecond?

Reply:
Arguments are in milliseconds, so it updates every second.  The reason is we don't want to burn CPU trying to react to a 1-millisecond interval.

Reply:
I would recommend changing the 0n and 1000n to be named constants like NO_DELAY and ONE_SECOND for readability.